src/rcpp_constants.cpp

```diff
@@ -881,10 +882,30 @@ return D_Q10_RH;
 }
 
 //' @describeIn parameters CO2 fertilization factor (\code{"(unitless)"})
+//' @param biome Biome for which to retrieve parameter. If missing or
+//'   `""`, default to `"global"`.
 //' @export
 // [[Rcpp::export]]
-String BETA() {
-return D_BETA;
+String BETA(String biome = "") {
+  if (biome == "") return D_BETA;
+  // `Rcpp::String` has a `+=` method, but no `+` method, so have to use
+  // this clunky workaround.
```

Comment:
I originally had this as `std::string`, which apparently Rcpp supports directly (translates to `String`?), precisely because it supports the `+` operator. I changed it to `Rcpp::String` for consistency with the other calls, but perhaps we can change to `std::string` here and everywhere else?

Reply:
I suspect that what's happening is that `Rcpp::string` has a constructor for initializing with a `std::string`, and vice versa, so it doesn't matter which one you use, since it will get converted as necessary.  If that's the case, then I'd say use whichever one you like.  I probably used `Rcpp::string` in the other functions because I never thought to check whether a `std::string` would work.

Reply:
For consistency and (IMHO) clarity, I'm gonna leave this as `Rcpp::String`. Marking this thread as resolved.